src/etl/ClinicalTrialModelFactory.test.ts

```diff
@@ -0,0 +1,586 @@
+import { ClinicalTrialModelFactory } from './ClinicalTrialModelFactory'
+import { ClinicalTrialModel } from '../shared/models/ClinicalTrialModel'
+import { riphCtisDto, riphDmDto, riphJardeDto1 } from '../shared/test/helpers/elasticsearchHelper'
+
+describe('clinical trial model factory', () => {
+  const unavailable = 'INDISPONIBLE'
+
+  describe('ctis', () => {
```

Comment:
peaufinage (non bloquant): Pour la compréhension néophyte, j'aurai tendance à proposer :

```suggestion
  describe('When building a CTIS clinical trial model', () => {
```
ou 
```suggestion
  describe('# CTIS clinical trial model', () => {
```